src/cascade/executor/epiviz_runner.py

```diff
@@ -99,8 +101,23 @@ def add_omega_constraint(model_context, execution_context, sex_id):
         max_time = np.max(list(model_context.input_data.times))  # noqa: F841
         # The % 5 is to exclude annual data points.
         asdr = asdr.query("time_lower >= @min_time and time_upper <= @max_time and time_lower % 5 == 0")
-    model_context.rates.omega.parent_smooth = build_constraint(asdr)
-    MATHLOG.debug(f"Add {asdr.shape[0]} omega constraints from age-standardized death rate data.")
+
+    parent_asdr = asdr[asdr.node_id == model_context.parameters.location_id]
+    model_context.rates.omega.parent_smooth = build_constraint(parent_asdr)
+    MATHLOG.debug(f"Add {parent_asdr.shape[0]} omega constraints from age-standardized death rate data to the parent.")
+
+    location_hierarchy = get_location_hierarchy_from_gbd(execution_context)
+    location = location_hierarchy.get_node_by_id(execution_context.parameters.location_id)
+    children = {d.id for d in location.children}  # noqa: F841
+    children_asdr = asdr.query("node_id in @children")
+    model_context.rates.omega.child_smoothings = [
+        (node_id, build_constraint(child_asdr))
+        for node_id, child_asdr in children_asdr.groupby('node_id')
+    ]
```

Comment:
This is succinct for calling build_constraint many times.

Reply:
Please note that I didn't do it all on one line! I'm growing.